gen2-social-distancing/requirements.txt

```diff
@@ -1,5 +1,5 @@
 imutils==0.5.3
 concurrent-log-handler==0.9.16
 opencv-python==4.5.1.48
-depthai==2.7.2.0
+depthai==2.10.0.0
```

Comment:
If gen2-social-distancing throws error, why would you update it?

Reply:
Because it throws an error in both versions, so I suspect the depthai version isn't the problem.

Reply:
so.. will you try to find the root cause of the problem?